### PR TITLE
Add common doc for jsonapi pagination

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: docs/jsonapi.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[Table of contents](README.md#table-of-contents)
+
+# JSON-API
+
+## Introduction
+
+Except for the routes in `/data`, which imitates couchdb, most of the stack exposes a JSON-API interface.
```
Comment (nono) [inline docs/jsonapi.md]: s/imitates/imitate/

File: docs/jsonapi.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+See [JSON-API specification](http://jsonapi.org/format/) for more information.
+
+
+
+## Pagination
+
+All routes that returns a list are (or will be) paginated.
```
Comment (nono) [inline docs/jsonapi.md]: s/returns/return/

File: docs/jsonapi.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+As recommended for couchdb, we use **cursor-based** pagination.
+
+The default page limit is determined on a by-route basis. The client can require a lower limit using `page[limit]` query parameter. If the client does not specify a limit or if the limit is higher than default, default limit will be used instead.
```
Comment (nono) [inline docs/jsonapi.md]: I think there are some cases where we may authorize to put an higher limit than the default

**Comment by aenario** [comment]
replaced by #440 with @nono comments take.

**Pull request closed without merging**